src/playback/internal/playbackcontroller.cpp

```diff
@@ -1434,7 +1434,19 @@ MeasureBeat PlaybackController::currentBeat() const
 
 msecs_t PlaybackController::beatToMilliseconds(int measureIndex, int beatIndex) const
 {
-    return notationPlayback() ? tickToMsecs(notationPlayback()->beatToTick(measureIndex, beatIndex)) : 0;
+    return notationPlayback() ? playedTickToMsecs(notationPlayback()->beatToRawTick(measureIndex, beatIndex)) : 0;
```

Comment:
I think this should get the same treatment as `beatToSecs`. `beatToRawTick` returns raw ticks, but `playedTickToMsecs` expects played ticks. 
(This bug already existed before your changes, but since the fix should be easy, it would be awesome to include that in this PR.)